### PR TITLE
[DS-723] Integrate a functionality of the CKEditor Bootstrap Buttons module to Editor Advanced Link module

### PR DESCRIPTION
**Pull request by aleevas**
Original Issue, this PR is going to fix: [DS-723](https://yusa.atlassian.net/browse/DS-723)

## Steps for review

- [ ] Log as admin
- [ ] go to /admin/config/content/formats/manage/full_html
- [ ] scroll down to **Advanced links** settings
- [ ] enable a new option **Bootstrap Button** and save the form
- [ ] go to /node/add/blog
- [ ] to content area insert a link via link  WYSIWYG widget
- [ ] verify that you can see a new foldable section **Bootstrap Button**
- [ ] add values for Bootstrap Button (Instruction you can find here /admin/help/editor_advanced_link)
- [ ] add value for **CSS classes** field in Advanced section
- [ ] save the node
- [ ] verify that button has bootstrap style
- [ ] open DevTools on that link
- [ ] verify that link has value from the  **CSS classes**  field of Advanced section

BootstrapButtonsOption
EditLinkWithBB

## General checks
- [ ] All coding styles are fulfilled and there are no any issues reported by CodeSniffer. See [Code of Conduct](https://github.com/ymcatwincities/openy/wiki/Open-Y-Code-of-Conduct-and-Best-Practices).
- [ ] Try to use Conventional commit messages accordingly to the standard https://www.conventionalcommits.org/en/v1.0.0/#specification
- [ ] [Documentation](https://github.com/ycloudyusa/yusaopeny/tree/main/docs) has been updated according to PR changes.
- [ ] [Steps for review](https://github.com/ymcatwincities/openy/pull/94#issue-204580200) have been provided according to PR changes. <br/>Steps for review
- [ ] Make sure you've provided all necessary hook\_update\_N to [support upgrade path](https://github.com/ycloudyusa/yusaopeny/blob/main/docs/Development/Upgrade%20path.md).
- [ ] Make sure your git email is associated with an account on drupal.org, otherwise, you won't get commits there. <br/>drupal.org email
- [ ] If you would like to get credits on drupal.org, [check documentation](https://github.com/ycloudyusa/yusaopeny/blob/main/docs/Development/Contributing.md#drupalorg-credits).

Thank you for your contribution!

### CODE REVIEW

**Review by rollki — APPROVED** [review]
LGTM.

**Comment by podarok** [comment]
ok to test

File: composer.json [hunk 1]
```diff
@@ -394,6 +394,9 @@
             },
             "drupal/responsive_favicons": {
                 "Issue #3376766: Incorrect path if Drupal is installed in subfolder": "https://git.drupalcode.org/project/responsive_favicons/-/merge_requests/6.diff"
+            },
+          "drupal/editor_advanced_link": {
+                "Issue #3423208: Integrate a functionality of the CKEditor Bootstrap Buttons module": "https://git.drupalcode.org/project/editor_advanced_link/-/merge_requests/19.diff"
```
Comment (podarok) [inline composer.json:399]: MR is in DRAFT, we can't let it in @aleevas @rollki 
Could you make it Ready?

**Review by podarok — CHANGES_REQUESTED** [review]
we need to have MR at least in RTBC state @aleevas @rollki

File: composer.json (hunk 1 above)
Comment (rollki) [reply composer.json:399]: Hi @podarok 
Of course, thank you!

Comment (rollki) [reply composer.json:399]: Also, I think we need to use a static patch here and not a diff merge request, what do you think?

Comment (rollki) [reply composer.json:399]: cc @aleevas

**Comment by rollki** [comment]
ok to test

**Comment by rollki** [comment]
ok to test

**Comment by podarok** [comment]
retest this please

**Comment by podarok** [comment]
We can't have builds from forks @aleevas @rollki 
Code should be pushed to feature branch

**Comment by podarok** [comment]
continue here https://github.com/YCloudYUSA/yusaopeny/pull/161 @aleevas @rollki

**Pull request merged**